crates/esthri-cli/src/main.rs

```diff
@@ -126,6 +127,9 @@ enum EsthriCommand {
         /// The key name of the object (example: a/key/name.bin)
         #[structopt(long)]
         key: String,
+        /// The storage class of the object (example: RRS)
+        #[structopt(long = "storage", default_value = "STANDARDIA")]
+        storage_class: String,
```

Comment:
`clap` automatically supports types that implement "FromStr", so instead of a `String` here and parsing it later, you can do

```rust
#[structopt(long = "storage", default_value = "S3StorageClass::StandardIA")]
storage_class: S3StorageClass,
```